EventFilter/EcalRawToDigi/plugins/EcalRawToDigiGPU.cc

```diff
@@ -62,6 +64,7 @@ EcalRawToDigiGPU::EcalRawToDigiGPU(const edm::ParameterSet& ps)
       digisEBToken_{produces<OutputProduct>(ps.getParameter<std::string>("digisLabelEB"))},
       digisEEToken_{produces<OutputProduct>(ps.getParameter<std::string>("digisLabelEE"))},
       eMappingToken_{esConsumes<ecal::raw::ElectronicsMappingGPU, EcalMappingElectronicsRcd>()},
+      nbytes_per_fed_max_{ps.getParameter<uint32_t>("nbytes_per_fed_max")},
```

Comment:
could you change `nbytes_per_fed_max` to `maxFedSize` for consistency with the other options ?